### PR TITLE
Revert/#283

### PR DESCRIPTION
**Pull request by oblm**
Reverts: https://github.com/spatialos/UnrealGDKExampleProject/pull/283/

We were forced to pick a [different approach](https://improbableio.atlassian.net/browse/UNR-4903?focusedCommentId=451317).

The `circle ci pipeline check` output for this change is:

```ERRO Pipeline not as expected, would apply the following diff:
     @@ -4,10 +4,10 @@
      ID: ""
      Name: unrealgdkexampleproject-nightly
      ProviderSettings:
     -  branch_configuration: '!dry-run/* !skip-ci/* !doc/* !docs/*'
     +  branch_configuration: ""
        publish_commit_status_per_step: true
     -  pull_request_branch_filter_configuration: '!dry-run/* !skip-ci/* !doc/* !docs/*'
     -  pull_request_branch_filter_enabled: true
     +  pull_request_branch_filter_configuration: ""
     +  pull_request_branch_filter_enabled: false
      Repository:
        url: git@github.com:spatialos/UnrealGDKExampleProject.git
      Slug: ""
     `

### CODE REVIEW

**Review by simonsarginson — APPROVED** [review]

File: .buildkite/nightly.steps.yaml [hunk 1]
```diff
@@ -11,8 +11,8 @@ steps:
       message: "exampleproject-4.26 ${BUILDKITE_MESSAGE}"
       env: 
         BUILD_TYPE: "EXAMPLE"
-        GDK_BRANCH: "${GDK_BRANCH:-match_branch_name_pref::master}"
```
Comment (simonsarginson) [inline .buildkite/nightly.steps.yaml:14]: Looks like you caught unrelated changes in this revert.

Comment (oblm) [reply .buildkite/nightly.steps.yaml:14]: Good spot, will do this change again.

Comment (oblm) [reply .buildkite/nightly.steps.yaml:14]: https://github.com/spatialos/UnrealGDKExampleProject/pull/289
